superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.test.tsx

```diff
@@ -1325,23 +1332,23 @@ describe('DatabaseModal', () => {
       ...jest.requireActual('src/views/CRUD/hooks'),
       useSingleViewResource: jest.fn(),
     }));
-    const useSingleViewResourceMock = jest.spyOn(
-      hooks,
-      'useSingleViewResource',
-    );
-
-    useSingleViewResourceMock.mockReturnValue({
-      state: {
-        loading: false,
-        resource: null,
-        error: { _schema: 'Test Error With Object' },
-      },
-      fetchResource: jest.fn(),
-      createResource: jest.fn(),
-      updateResource: jest.fn(),
-      clearError: jest.fn(),
-      setResource: jest.fn(),
-    });
+    // const useSingleViewResourceMock = jest.spyOn(
```

Comment:
Should we remove these then?

Reply:
agree with Antonio

Reply:
These are from a todo from long ago that I forgot about, we can definitely remove these and come back to it another time 😅

Reply:
good call